core/platform/auth/firebase_auth_services.py

```
@@ -68,7 +68,7 @@
 import webapp2
 from firebase_admin import auth as firebase_auth
 from firebase_admin import exceptions as firebase_exceptions
-from typing import List, Optional
+from typing import Any, List, Optional
```

Ruff (0.15.17)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)